server/src/immich/controllers/face.controller.ts

```diff
@@ -18,11 +18,16 @@ export class FaceController {
   }
 
   @Put(':id')
-  reassignFacesById(
-    @Auth() auth: AuthDto,
+  reassignFace(
```

Comment:
Is this endpoint used in the UI anywhere? What is it's purpose?

Reply:
This is used when you assign a face from one person to another on the person-side-panel

Reply:
I did not find it in the code can you link it for me?

Reply:
https://github.com/immich-app/immich/pull/5511/files#diff-0748192911c047330e6448297fb246113a246e8fb459d6ba0af512387e7abf0dR242-R245